### PR TITLE
Added sourcery config file

### PR DESCRIPTION
**Pull request by SiddhantSadangi**
Signed-off-by: Siddhant Sadangi <siddhant.sadangi@gmail.com>

# Description

Added `.sourcery.yaml`

__Related to:__ `(ClickUp/JIRA task name)`

---

## ❔ This change

- [ ] adds a new feature
- [ ] fixes breaking code
- [X] is cosmetic (refactoring/reformatting)

---

## ✔️ Pre-merge checklist

Mark which items have been completed:

- [ ] Refactored code ([sourcery](https://sourcery.ai/))
- [ ] Tested code locally
- [ ] Added code to GitHub tests ([notebooks](workflows/test-notebooks.yml), [scripts](workflows/test-scripts.yml))
- [ ] Updated GitHub [README](../README.md)

### 🪝 Run the following to both [pre-commit](https://pre-commit.com/) hooks and JupyterLab (turn on `format on save`)

- [ ] Stripped outputs from notebooks - ([nbstripout](https://pypi.org/project/nbstripout/))
- [ ] Sorted imports - ([isort](https://pycqa.github.io/isort/))
- [ ] Formatted code - ([back](https://github.com/psf/black))

---

## 🧪 Test Configuration

- OS:
- Python version:
- Neptune version:
- Affected libraries with version:

### CODE REVIEW

File: .sourcery.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#   sourcery_branch: sourcery/{base_branch}
+
+# clone_detection:
+#   min_lines: 3
+#   min_duplicates: 2
+#   identical_clones_only: false
+
+# proxy:
+#   url:
+#   ssl_certs_file:
```
Comment (Blaizzy) [inline .sourcery.yaml:66]: Can we remove these comments? If not, why?

**Review by Blaizzy — CHANGES_REQUESTED** [review]

File: .sourcery.yaml (hunk 1 above)
Comment (SiddhantSadangi) [reply .sourcery.yaml:66]: we can.
i didn't because we might want to use some of these rules in the future

Comment (Blaizzy) [reply .sourcery.yaml:66]: hmm... Ok 👍

**Review by Blaizzy — APPROVED** [review]
LGTM!

**Pull request merged**